tests/tests/tests.py

```diff
@@ -244,3 +244,14 @@ def test_works_with_create_view_post(self):
         resp = viewfn(request)
         self.assertEqual(resp.status_code, 302)
         self.assertEqual(Badge.objects.count(), 2)
+
+    def test_non_field_errors_with_formset(self):
+        form = BookMultiForm({
+            'book-name': '',
+            'images-0-name': '',
+            'images-TOTAL_FORMS': '3',
+            'images-INITIAL_FORMS': '0',
+            'images-MAX_NUM_FORMS': '1000',
+        })
+        # assertDoesntRaise AttributeError
+        self.assertEqual(form.non_field_errors().as_text(), '* It broke')
```

Comment:
https://github.com/fusionbox/django-betterforms/blob/formsets-non_field_errors/tests/tests/forms.py#L45

The asterisk is added by the `as_text` method